modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientExperimentalCodegen.java

```diff
@@ -95,6 +95,9 @@ public PythonClientExperimentalCodegen() {
         modelTestTemplateFiles.remove("model_test.mustache", ".py");
         modelTestTemplateFiles.put("python-experimental/model_test.mustache", ".py");
 
+        apiTestTemplateFiles.remove("api_test.mustache", ".py");
+        apiTestTemplateFiles.put("python-experimental/api_test.mustache", ".py");
```

Comment:
It does not look like we are changing the api_test template.
Can we remove this?